_docs/organizations-and-collections.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Organizations and Collections
+permalink: /docs/publisher-tutorials/organizations-and-collections/
+---
+
+# Organizations and Collections
+
+## Organizations
+Organizations are landing pages for collaborations, institutions, consortiums, companies, etc. that allow users to showcase tools and workflows. This is achieved through the creation of collections, which are groupings of related tools and workflows. The users of an organization do not need to own the tools or workflows in any way, the tools and workflows just have to be published. Collections can be thought of as a playlist on a Music Streaming Service where tools and workflows are analogous to individual songs. They can be shared publicly, and the user does not need to own them.
```

Comment:
>The users of an organization do not need to own the tools or workflows in any way, the tools and workflows just have to be published.

This is a run-on sentence. I would either put a semi-colon or a period to separate them.


I would have `music streaming service` -- only have the capitals if it were the name of an actual company.

Reply:
Originally it was gonna be a company so I used capitals to stand out, but I think it is better not to mention a company.